WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsTableHeaderView.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import DesignSystem
+
+final class NotificationsTableHeaderView: UITableViewHeaderFooterView {
+
+    static let reuseIdentifier: String = String(describing: NotificationsTableHeaderView.self)
+
+    // MARK: - Properties
+
+    var text: String? {
+        didSet {
+            self.update(text: text)
+        }
+    }
+
+    // MARK: - Init
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+        self.setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Setup
+
+    private func setup() {
+        self.contentConfiguration = {
+            var config = super.defaultContentConfiguration()
+            config.textProperties.font = Appearance.textFont
+            config.textProperties.color = Appearance.textColor
+            config.directionalLayoutMargins = Appearance.layoutMargins
+            return config
+        }()
+        if #available(iOS 16.0, *) {
+            self.backgroundConfiguration = {
+                var config = self.defaultBackgroundConfiguration()
+                config.backgroundColor = Appearance.backgroundColor
+                config.visualEffect = nil
+                return config
+            }()
+        } else {
+            self.contentView.backgroundColor = Appearance.backgroundColor
+        }
+    }
+
+    // MARK: - Update
+
+    private func update(text: String?) {
+        guard var config = contentConfiguration as? UIListContentConfiguration else {
+            return
+        }
+        config.text = text
+        self.contentConfiguration = config
+    }
+
+    // MARK: - Constants
+
+    private enum Appearance {
+        static let backgroundColor = UIColor.systemBackground
+        static let textColor = UIColor.DS.Foreground.primary ?? .text
+        static let textFont = WPStyleGuide.fontForTextStyle(.headline, fontWeight: .semibold)
```

Comment:
We can use the Design System extension instead. It is an extension on UILabel().setStyle(.headline). You won't have to set the weight as the function tackles that by itself.

However, if this is for the "Today" bit, the font should be `.bodyLarge(.emphasized)` and not `.headline`

Reply:
The font `.bodyLarge(.emphasized)` has 15pt in size, while on Figma it has 17pt, that's why I chose `.headline` to match the size.

| Design System | Figma |
| -------------- | ------- |
| ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/d183995a-edf5-4670-ad39-d374a420d91e) | ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/e96461ad-9b6e-4c8e-a96c-c4cb2544f82e) |

Reply:
You're right. There was a mistake in the UIKit extensions of Design System and it was fixed with https://github.com/wordpress-mobile/WordPress-iOS/pull/22561.

So merging `trunk` in should resolve it.